scorecards.go

```diff
@@ -0,0 +1,130 @@
+package opslevel
+
+import (
+	"fmt"
+)
+
+type ScorecardId struct {
+	Aliases []string `graphql:"aliases"`
+	Id      ID       `graphql:"id"`
+}
+
+type Scorecard struct {
+	ScorecardId
+
+	Description   string      `graphql:"description"`
+	Filter        Filter      `graphql:"filter"`
+	Name          string      `graphql:"name"`
+	Owner         EntityOwner `graphql:"owner"`
+	PassingChecks int         `graphql:"passingChecks"`
+	ServiceCount  int         `graphql:"serviceCount"`
+	TotalChecks   int         `graphql:"totalChecks"`
```

Comment:
What do you think about renaming this to `ChecksCount` to match `ServiceCount` ?